src/main/ipv-stub/ipv-authorize.ts

```diff
@@ -15,38 +18,119 @@ export const handler: Handler = async (
     case "POST":
       return post(event);
     default:
-      logger.error("Method not allowed");
-      return {
-        statusCode: 405,
-        body: JSON.stringify({
-          message: "Method not allowed",
-        }),
-      };
+      return errorResponse(405, "Method not allowed");
   }
 };
 
-function get(_event: APIGatewayProxyEvent): APIGatewayProxyResult {
+async function get(
+  event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> {
   logger.info("Reached the GET endpoint");
```

Comment:
Can improve this log message. "IPV authorize Get endpoint" something like that.

Reply:
I think we can remove it altogether now the endpoint does something? This was just the hello world implementation I think